pkg/defaults.py

```diff
@@ -1,8 +1,8 @@
 OPTS.update(
     name = VAR.fullname,
     url = 'https://github.com/sociomantic/dhtnode',
-    maintainer = 'Sociomantic Labs GmbH <tsunami@sociomantic.com>',
-    vendor = 'Sociomantic Labs GmbH',
+    maintainer = 'dunnhumby Germany GmbH <tsunami@sociomantic.com>',
```

Comment:
Eventually I think we'll need to change the e-mail, but that's for the future I guess...